neuroinformatics/hbp_prov/hbp-prov-schema-v3.1.json

```diff
@@ -0,0 +1,959 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "description": "schema for a the json that describes the registration activity.",
+  "version": "3.0",
```

Comment:
version number is 3.1, right? Also, let's document our naming policy, and policy of creating the new files versus making changes to the existing ones. This 3.1 version as a separate file looks strange and even more so if it's backwards-compatible.